### DISCUSSION TITLE
How to Configure API Key for ASNmap.

### DISCUSSION THREAD

**Discussion by imAneeshD** [projectdiscovery/httpx#1646] [Q&A] [answered]
I have the API key, where to add it, so that I dont get "[asnmap-api] missing or invalid api key (get free api & configure it from https://cloud.projectdiscovery.io/?ref=api_key)" ERROR.

**Comment by olearycrew** [comment, marked as answer]
`asnmap -auth {api-key}`

**Comment by imAneeshD** [comment]
Got it. Thank you for your response.
